src/pages/CalculatorPage.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Calculator from '../components/Calculator'; // eslint-disable-line
+
+const CalculatorPage = () => {
+  const calcPageStyle = {
+    display: 'flex',
+    flexDirection: 'row',
+    gap: '2rem',
+    justifyContent: 'space-between',
+  };
+
```

Comment:
- Please fix this mistake so that your website can be reviewed
- 
![Capture](https://user-images.githubusercontent.com/63972670/199870733-71308892-8b39-4e60-a59e-010a1e8ab08b.PNG)